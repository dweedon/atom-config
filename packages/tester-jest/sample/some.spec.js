test('string are equals', () => {
  expect('stringA').toBe('stringA');
});

it('snapshot test', () => {
  expect({ a: 1, b: 2 }).toMatchSnapshot();
});
