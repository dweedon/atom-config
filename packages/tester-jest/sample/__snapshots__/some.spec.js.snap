// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
Object {
  "a": 1,
  "b": 2,
}
`;
